nestkernel/nestmodule.cpp

```diff
@@ -475,6 +475,69 @@ NestModule::SimulateFunction::execute( SLIInterpreter* i ) const
 }
 
 /* BeginDocumentation
+   Name: Run - simulate n milliseconds
+
+   Synopsis:
+   n(int) Run -> -
+
+   Description: Simulate the network for n milliseconds.
+   Call prepare before, and cleanup after.
+   n simulate = prepare n/m run ... m times... cleanup
```

Comment:
Make `prepare`, `run` and `simulate` uppercase, wherever they refer to the corrresponding SLI commands. Those are case sensitive. This comment applies throughout this PR.